### PR TITLE
Handle contract to events

### PR DESCRIPTION
**Pull request by walterbucolo-eb**
DetailContract view was created to show contract data.
Now, one contract can be linked to more than one event. 
This events are listed in DetailContract view, where more events can be created as well.
event_id field was removed.

### CODE REVIEW

File: app/static/css/detail_contract.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    margin-top: 10px;
+}
+.margin-link {
+    margin-top: 15px;
+}
+.margin-delete {
+    margin-top: 4px;
+    margin-left: 15px;
+}
+table td {
```
Comment (leonardoa-eb) [inline app/static/css/detail_contract.css]: Take care of this kind of selector. You are affecting ALL the `tables td` 😱 . A better approach could create a class for these elements or at least create a class for the tables that you wants to modify.

File: app/templates/app/create_condition.html [hunk 2]
```diff
@@ -43,7 +43,7 @@ <h5 class="card-title">Installment</h5>
                   <h4>Check GTS/GTF</h4>
                   <form id="superset-link" action="">
                     <label for="event-id">Event id:</label>
-                    <input id="event-id" name="event-id" class="mr-4" type="" placeholder="Event id" value="{{ installment.contract.event_id }}">
+                    <input id="event-id" name="event-id" class="mr-4" type="" placeholder="Event id">
```
Comment (leonardoa-eb) [inline app/templates/app/create_condition.html]: Why you define inputs with `type` attribute blank? 
Based on the provided name, you could choose for `type='number'` or just put `type='text'`(and also you can put a validator pattern in HTML)

File: app/templates/app/detail_contract.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    {% endfor %}
+                    </div>
+            {% else %}
+                <p class="lead indent"><i>There are no events for this contract yet.</i></p>
+            {% endif%}
+            <div class="lead indent-form">
+                <form action="{% url 'events-create' contract.id %}" method="POST">{% csrf_token %}
+                    <div class="form-group row">
+                        <div class="col-sm-3">
+                            <input id="event_name" type="text" class="form-control" name="Event Name" value="" placeholder="Event name">
```
Comment (leonardoa-eb) [inline app/templates/app/detail_contract.html]: Why the value attribute is blank?

File: app/templates/app/detail_contract.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                        </div>
+                        <div class="col-sm-3">
+                            <input id="event_id" class="form-control" type="text" name="Event id" value="" placeholder="Event id">
```
Comment (leonardoa-eb) [inline app/templates/app/detail_contract.html]: Same

File: app/templates/app/detail_contract.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                </form>
+            </div>
+        </div>
+        <a href="{% url 'contracts' %}" role="button" class="btn btn-outline-secondary"><i class="far fa-arrow-alt-circle-left"></i>  Back</a>
+    </div>
+    <script>
+        $('.delete-file-form').submit(e => {
+            e.preventDefault();
+            if (confirm('Confirm delete for this event')) e.target.submit();
+        });
```
Comment (leonardoa-eb) [inline app/templates/app/detail_contract.html]: Sounds like that you start this PR from a old branch...

File: app/tests.py [hunk 6]
```diff
@@ -26,7 +26,7 @@
     ContractFactory,
     InstallmentConditionFactory,
     InstallmentFactory,
-)
+    EventFactory)
```
Comment (leonardoa-eb) [inline app/tests.py]: Nit put that brace in a new line. You can define this kind of automatization in this settings https://www.jetbrains.com/help/pycharm-edu/configuring-code-style.html

File: app/views.py [hunk 7]
```diff
@@ -98,7 +100,7 @@ class Meta:
 class ContractUpdate(UpdateView):
     template_name = "app/update_contract.html"
     model = Contract
-    fields = ["organizer_account_name", "organizer_email", "signed_date", "event_id", "user_id"]
+    fields = ["organizer_account_name", "organizer_email", "signed_date", "user_id"]
```
Comment (leonardoa-eb) [inline app/views.py]: A general comment. What's the code style for this project or for your team? Bc I'm seeing different usages of single comma and double comma across the whole project. One way to fix this is to automate these tasks with a tool/library. 
Black is a format tool that you can add to your project. The idea behind this is hit and run. Black will we in charge of auto-format your file/folder for you and you don't pay attention for code styling.
https://github.com/psf/black

(Also you can configure this using a `pre-commit` tool https://pre-commit.com/)

Comment (walterbucolo-eb) [reply app/views.py]: We don't have defined code styles. Is a very good idea use this tool. Thanks.

File: app/templates/app/detail_contract.html (hunk 3 above)
Comment (walterbucolo-eb) [reply app/templates/app/detail_contract.html]: i made a mistake.

File: app/templates/app/detail_contract.html (hunk 5 above)
Comment (walterbucolo-eb) [reply app/templates/app/detail_contract.html]: Yes, i forgot delete this code.

**Pull request merged**